.storybook/storybook-data.js

```diff
@@ -66,71 +66,6 @@ module.exports = [
 			},
 		},
 	},
-	{
-		id: 'global-notifications--small-with-image',
-		kind: 'Global/Notifications',
-		name: 'Small with Image',
-		story: 'Small with Image',
-		parameters: {
-			fileName: './stories/notifications.stories.js',
-			options: {
-				hierarchyRootSeparator: '|',
-				hierarchySeparator: {},
-			},
-		},
-	},
-	{
-		id: 'global-notifications--small-with-no-image',
```

Comment:
> The V6 story migration is a clear improvement (and nice to see the additional stories that you've added), however the VRT scenarios from here onward have been removed with no replacement...

I left one scenario for the banner, other scenarios are useless and just waste our time and resources.

> I'd suggest adding scenarios to `ReminderBanner.stories.js` and `UserInputSuccessBannerNotification.stories.js` to restore the coverage, as we do have a separate [issue](https://github.com/google/site-kit-wp/issues/7388) where we can take stock of the VRT suite with a view to determining unneeded stories to trim.

Returned the scenario for the user input banner since it existed before in the legacy stories. As to the reminder banner, i haven't added it because we don't have vrt tests for that banner in develop, plus that banner will no longer be needed soon.